ibridges/irodsconnector/permissions.py

```diff
@@ -47,8 +47,9 @@ def set(self, perm: str, user: Optional[str] = None, zone: Optional[str] = None,
             user = self.session.username
         if zone is None:
             zone = self.session.zone
-        if perm == "null" and user == self.session.username and zone == self.session.zone:
-            raise ValueError("Cannot set your own permissions to null, because you would lose "
+        if perm not in ["inherit", "noinherit"] and user == self.session.username and \
```

Comment:
Maybe you can leave a small comment on the "inherit"/"noinherit" or is it obvious?

Reply:
Done!